python/sglang/srt/managers/scheduler.py

```diff
@@ -629,11 +629,22 @@ def handle_generate_request(
 
         # Truncate prompts that are too long
         if len(req.origin_input_ids) > self.max_req_input_len:
-            logger.warning(
-                "Request length is longer than the KV cache pool size or "
-                "the max context length. Truncated!!!"
-            )
-            req.origin_input_ids = req.origin_input_ids[: self.max_req_input_len]
+            if self.server_args.allow_auto_truncate:
+                logger.warning(
+                    "Request length is longer than the KV cache pool size or "
+                    "the max context length. Truncated!!!"
```

Comment:
Can you also print the two lengths like this?
https://github.com/sgl-project/sglang/blob/78e974b2a5c890ae3b05c0700d1a82c2b9e6814e/python/sglang/srt/managers/scheduler.py#L696-L698

Reply:
Rebased main branch and updated with the latest msg.